src/pages/blogpostpage/blogpostpage.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from 'react';
+import data from '../data/posts.json';
+import {useHistory} from "react-router-dom";
+
+
+
+function BlogPostPage() {
+    const history = useHistory();
+    function HandleClick() {
+        console.log("Blog inhoud lezen");
+        history.push('./blogposts');
+    }
+
+   const [posts, setPosts] = useState([]);
+    console.log(data)
+
+   useEffect( () => {
+       setPosts(data);
+   }, [])
```

Comment:
useState en useEffect zijn niet nodig, we halen uit een locale database, en niet van een andere server of api, we hebben geen state nodig maar een enkel post object uit de database.